app/common_handlers.py

```diff
@@ -269,9 +252,9 @@ async def get(self, request):
         self.log_entry(request, display_region + '/' + user_journey + '/' + sub_user_journey + '/enter-address')
 
         if user_journey == 'start':
-            await check_permission(request)
-
-        session = await get_session(request)
+            session = await check_permission(request)
+        else:
+            session = await get_session(request)
```

Comment:
seems to me like this should be `get_existing_session`. ?

Reply:
There may not be an existing session in the CommonEnterAddress view for the user journeys:
- /request/access-code/enter-address/ ( request a new household or individual access code from the Start page)
- /request/paper-questionnaire/enter-address/
- /request/continuation-questionnaire/enter-address/
I have therefore retained the use of the aiohttp_session get_session function to return a new session if one doesn't exist. This is the beginning of the user journey for these fulfillments.

Reply:
ok fair enough